packages/calcite-components/src/components/block/block.stories.ts

```diff
@@ -215,3 +215,31 @@ export const contentSpacing_TestOnly = (): string =>
       <div>Some text that has padding built in</div>
     </calcite-block>
   `;
+
+export const loadingCaseLoadingWithSlottedIcon_TestOnly = (): string =>
```

Comment:
I think we can drop `loadingCase` since the name indicates it's loading-related and `_TestOnly` reflects a test case.